src/overlays/actors/ovl_Bg_Hidan_Rsekizou/z_bg_hidan_rsekizou.c

```diff
@@ -21,13 +20,193 @@ const ActorInit Bg_Hidan_Rsekizou_InitVars = {
     (ActorFunc)BgHidanRsekizou_Update,
     (ActorFunc)BgHidanRsekizou_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Hidan_Rsekizou/BgHidanRsekizou_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Hidan_Rsekizou/BgHidanRsekizou_Destroy.s")
+static ColliderJntSphItemInit sJntSphItemsInit[6] = {
+    {
+        { 0x00, { 0x20000000, 0x01, 0x04 }, { 0x00000000, 0x00, 0x00 }, 0x19, 0x00, 0x00 },
+        { 1, { { 0, 30, 40 }, 25 }, 100 },
+    },
+    {
+        { 0x00, { 0x20000000, 0x01, 0x04 }, { 0x00000000, 0x00, 0x00 }, 0x19, 0x00, 0x00 },
+        { 1, { { -35, 32, 77 }, 32 }, 100 },
+    },
+    {
+        { 0x00, { 0x20000000, 0x01, 0x04 }, { 0x00000000, 0x00, 0x00 }, 0x19, 0x00, 0x00 },
+        { 1, { { -80, 35, 130 }, 42 }, 100 },
+    },
+    {
+        { 0x00, { 0x20000000, 0x01, 0x04 }, { 0x00000000, 0x00, 0x00 }, 0x19, 0x00, 0x00 },
+        { 1, { { 0, 30, -40 }, 25 }, 100 },
+    },
+    {
+        { 0x00, { 0x20000000, 0x01, 0x04 }, { 0x00000000, 0x00, 0x00 }, 0x19, 0x00, 0x00 },
+        { 1, { { 35, 32, -77 }, 32 }, 100 },
+    },
+    {
+        { 0x00, { 0x20000000, 0x01, 0x04 }, { 0x00000000, 0x00, 0x00 }, 0x19, 0x00, 0x00 },
+        { 1, { { 80, 35, -130 }, 42 }, 100 },
+    },
+};
+
+static ColliderJntSphInit sJntSphInit = {
+    { COLTYPE_UNK10, 0x11, 0x00, 0x00, 0x20, COLSHAPE_JNTSPH },
+    6,
+    sJntSphItemsInit,
+};
+
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 400, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneForward, 1500, ICHAIN_STOP),
+};
+
+static UNK_TYPE D_8088CD74[] = { 0x06015D20, 0x06016120, 0x06016520, 0x06016920,
```

Comment:
UNK_PTR is the correct type here. Warnings will be raised because the segment addresses are hardcoded instead of being symbols, but that will change eventually so you can just ignore them.
```suggestion
static UNK_PTR D_8088CD74[] = { 0x06015D20, 0x06016120, 0x06016520, 0x06016920,
```